net.sourceforge.qtpfsgui.LuminanceHDR.desktop

```diff
@@ -0,0 +1,25 @@
+[Desktop Entry]
```

Comment:
This should be actually kept in code repository so all distributions can benefit.

Reply:
This will be since the next application release.